libs/core/src/lib/multi-input/multi-input.component.ts

```diff
@@ -218,6 +227,19 @@ export class MultiInputComponent
     @Input()
     autoComplete = true;
 
+    /**
+     * Message announced by screen reader, when search suggestions opens.
+     */
+    @Input()
+    searchSuggestionMessage = $localize`:@@coreMultiInputSuggestion: suggestion found`;
```

Comment:
We're not using `@angular/localize` in `core`, use `@Input()` with raw strings instead for passing aria labels.

Reply:
@input variable is not required since the we need more than 3-4 variable to configure the a11y strings i think current solution is feasible, instead of creating unwanted @input variables